src/frontend/src/sass/_sideNav.scss

```diff
@@ -54,5 +56,13 @@
     .cursor-pointer {
         cursor:pointer;
     }
+
+    .bcgovbtn {

```

Comment:
Should this be in the `_buttons.scss`?

Reply:
I wanted to override the properties of the `bcgovbtn` in the `buttons.scss`

Would it still make sense to put it inside _buttons.scss?

Reply:
This overrides those properties everywhere.

Reply:
no, it will change only the `bcgovbtn` elements inside the `.sidebar`

Reply:
like yes for all `bcgovbtn` elements inside `.sidebar`

Reply:
I would avoid overwriting bcgovbtn even though it is only applicable to sidenav. In my PR i have fixed this.

Reply:
to be enhanced in [UTOPIA-836](https://apps.itsm.gov.bc.ca/jira/browse/UTOPIA-836)